server/student.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+const StudentSchema = new Schema(
+  {
+    firstName: String,
```

Comment:
`firstName` and `lastName` are required, please add it to the schema as you are doing in the CourseSchema